### PR TITLE
Create .travis.yml

### PR DESCRIPTION
**Pull request by douglasduteil**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+ 
+language: python
+python: 3
+
+cache: pip
+install:
+  - mkvirtualenv cdtn-scripts
+  - pip3 install -r requirements.txt
+  
+script: pipenv run pylint --output-format=colorized search
```
[the remark this answers is not in the document]
Comment (adipasquale) [reply .travis.yml]: I don't think we can both use pipenv and mkvirtualenv, so I'll try and fix this. Can you plug travisci to this repository so I can see how it goes @douglasduteil ? thanks

**Comment by adipasquale** [comment]
I don't think we can both use pipenv and mkvirtualenv, so I'll try and fix this. 

Can you plug travisci to this repository so I can see how it goes @douglasduteil ? thanks

**Comment by adipasquale** [comment]
ping @douglasduteil can you plug TravisCI so we can see if it works ?

**Comment by douglasduteil** [comment]
:up: @adipasquale @revolunet 
I added a codestyling and a linting system.
I don't know what is the latest trendy linter/prettier for Python.
What you think about `pylama` and `flake8` ?

**Comment by adipasquale** [comment]
I'd say `flake8` is more common but I'm really not an expert, I've used linting in python only a few times in my life 🙄

**Comment by douglasduteil** [comment]
Alright.
How do you run your tests in the `kali_extractor` ?

**Comment by adipasquale** [comment]
simply `python3 kali_extractor/tests/dict_utils_tests.py`

screen shot 2019-01-23 at 11 01 33

**Comment by douglasduteil** [comment]
Hum...
Not working...
\see https://travis-ci.com/SocialGouv/kali_dumps_scripts/jobs/172353543#L331-L336

**Comment by douglasduteil** [comment]
I added a deployment process on tags.
If you tag a commit on Github it will deploy a image the image version of it on Docker Hub

**Comment by adipasquale** [comment]
@douglasduteil I fixed the build 🎉  tests and flake8 linting run and pass. I removed pylama for now.

Now I'm just wondering if we shouldn't use the docker images to run the tests and linting in the CI. For example at https://github.com/SocialGouv/kali_dumps_scripts/pull/1/files#diff-354f30a63fb0907d4ad57269548329e3R48 shouldn't we replace it with something like : 
`docker-compose run extractor python tests/dict_utils_tests.py`  ? That's what I wrote in the README at least.

**Comment by adipasquale** [comment]
@douglasduteil I'm merging this PR so I can work on top of it but I'd like your opinion on my previous comment.

**Comment by douglasduteil** [comment]
> Now I'm just wondering if we shouldn't use the docker images to run the tests and linting in the CI. For example at https://github.com/SocialGouv/kali_dumps_scripts/pull/1/files#diff-354f30a63fb0907d4ad57269548329e3R48 shouldn't we replace it with something like :
docker-compose run extractor python tests/dict_utils_tests.py ? That's what I wrote in the README at least.

Why not if it makes more senses for you.
Both works for me :)

**Pull request merged**